deepmd/pt/utils/tabulate.py

```
@@ -0,0 +1,971 @@
+            The excluded pairs of types which have no interaction with each other.
+            For example, `[[0, 1]]` means no interaction between type 0 and type 1.
+    activation_function
+            The activation function in the embedding net. Supported options are {"tanh","gelu"} in common.ACTIVATION_FN_DICT.
+    """
+
+    def __init__(
+        self,
+        descrpt,
+        neuron: list[int],
+        type_one_side: bool = False,
+        exclude_types: list[list[int]] = [],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)